app/api/jobs/channels/route.ts

```diff
@@ -0,0 +1,37 @@
+import combinePrivyAccountWithChannelStats from "@/lib/privy/combineAccountsWithStats";
+import sortChannels from "@/lib/sortChannels";
+import { stack } from "@/lib/stack/client";
+import getChannelStats from "@/lib/supabase/getChannelStats";
+import { NextRequest } from "next/server";
+
+export async function GET(req: NextRequest) {
+  const TOP_CHANNELS = 10;
+  const TIP_AMOUNT = 11100;
```

Comment:
let's make this a full angel number
```suggestion
  const TIP_AMOUNT = 11111;
```